ts/Core/Tooltip.ts

```diff
@@ -1773,18 +1795,17 @@ class Tooltip {
             renderer
         } = tooltip;
         if (outside && container && renderer) {
-            // Set container size to fit the tooltip
-            const { width, height, x, y } = tooltipLabel.getBBox();
+            // Set container size to fit the bounds
+            const { width, x } = tooltipLabel.getBBox();
             renderer.setSize(
-                width + x,
-                height + y,
+                scrollablePixelsX ? width + x : bounds.right - bounds.left,
+                bounds.bottom - bounds.top,
```

Comment:
Does the renderer have to cover the whole bounds? Which are effectively the whole viewport when `tooltip.outside` and the whole chart when not? The initial idea here was that the renderer doesn't have to be larger than the tooltip itself.